addon/components/tag-post-list.js

```diff
@@ -9,8 +13,22 @@ const topPostCount = 3;
  *   <TagPostList @tag={{tag}} />
  */
 export default class TagPostList extends Component {
-  get topPosts() {
-    return this.args.tag.posts.slice(0, topPostCount);
+  @service store;
+
+  // this can't be a getter because returning a `DS.PromiseArray` from a getter
+  // causes an infinte loop
+  @tracked topPosts;
+
+  constructor() {
+    super(...arguments);
+
+    let ids = this.args.tag.hasMany('posts').ids().slice(0, topPostCount);
+
+    let promise = Promise.all(ids.map(id => this.store.findRecord('content', id)))
```

Comment:
```suggestion
    let promise = Promise.all(ids.map(id => this.store.findRecord('content', id)));
```

Reply:
hmm... I'm surprised this didn't show up in the lint 🤔

Reply:
fixed 👍